app/src/main/res/layout/number_validation.xml

```diff
@@ -48,17 +48,25 @@
                 android:hint="@string/hint_validation_name"
                 android:inputType="textPersonName" />
 
-            <Spinner android:id="@+id/phone_cc"
-                android:layout_width="fill_parent"
-                android:layout_height="wrap_content"
-                android:spinnerMode="dialog"/>
```

Comment:
I noticed this only now. Any particular reason for changing the spinner mode?

Reply:
yes. if the countrycode is on the same line, the dropdown caret should be on the left, so the number can be read +12 3456789